Yoast/wordpress-seo
# Remove the functionality and tests regarding Twitter creator

Since this was the last metadata type for Twitter, remove the test file and deprecate all public functions in the class

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Since this was the last metadata type to be removed for Twitter, I also deprecated the 'general' public functions and deleted the integration test. 

## Test instructions

This PR can be tested by following these steps:

* Make sure the `twitter:creator` tag is no longer rendered in the frontend.
* See the deprecation notices in the frontend.
    * To make the notices appear, add the following to `wp-seo-main`:
```
add_action( 'init', function() {
WPSEO_Twitter::get_instance()->twitter();
} );
``` 

## UI changes

## Documentation

## Quality assurance


Fixes #
